userService/.env

```diff
@@ -0,0 +1,4 @@
+SERVER_PORT=9191
```

Comment:
Don't push the .env file. Instead create another pull request stating things to set up in readme file. Example - You can add a section in readme file which mentions that to start the service create a .env file and add SERVER_PORT=9191, etc. Also create a readme file giving general information about your service. This readme file should be within your service not the main readme.